join.py

```diff
@@ -0,0 +1,20 @@
+import itertools
+
+
+def join(*lists, sep='-'):
+    """
+    The function receives an unlimited number of lists and another parameter called sep (optional)
+    and returns one list consisting of all the lists received as parameters. If the sep parameter is provided,
```

Comment:
Notice that there is an indentation problem in the documentation, please fix it.